### PR TITLE
radare2: update to 5.5.0

### PR DESCRIPTION
**Pull request by Claudemirovsky**

### CODE REVIEW

File: packages/radare2/fix-arm64-nand.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff -Nuar radare2-5.5.0.orig/libr/anal/p/anal_arm_cs.c radare2-5.5.0.mod/libr/anal/p/anal_arm_cs.c
+--- radare2-5.5.0.orig/libr/anal/p/anal_arm_cs.c	2021-11-15 01:18:10.653378506 -0300
++++ radare2-5.5.0.mod/libr/anal/p/anal_arm_cs.c	2021-11-15 01:18:58.165378472 -0300
+@@ -1361,9 +1361,6 @@
+ 	case ARM64_INS_AND:
+ 		OPCALL("&");
+ 		break;
+-	case ARM64_INS_NAND:
+-		OPCALL_NEG("&");
+-		break;
```
Comment (buttaface) [inline packages/radare2/fix-arm64-nand.patch:10]: Why is this needed?

Comment (Claudemirovsky) [reply packages/radare2/fix-arm64-nand.patch]: this [was] needed to fix a error when compiling(`undeclared identifier 'ARM64_INS_NAND'`), but I replaced it with a patch that disables the use of capstone-release(v4.0.2) and uses the default capstone version (v5/next branch), then the error will not occur.

in radareorg/radare2@645835794b0013b52042d5762d98d03fcbf250f6, trufae fixed the error, but I think disabling the use of capstone-release is better for now, as it will use the default version in r2 and so will give less headaches in the future.

**Comment by Grimler91** [comment]
Thanks!
